utilities_for_test.go

```diff
@@ -77,10 +77,11 @@ func NewIntegrationTestEnv(t *testing.T) IntegrationTestEnv {
 	} else if os.Getenv("HEDERA_NETWORK") == "localhost" {
 		network := make(map[string]AccountID)
 		network["127.0.0.1:50213"] = AccountID{Account: 3}
-		network["127.0.0.1:50214"] = AccountID{Account: 4}
-		network["127.0.0.1:50215"] = AccountID{Account: 5}
+		// network["127.0.0.1:50214"] = AccountID{Account: 4}
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**suggestion:** ‏Is this commented out on purpose or it was missed to be uncommented? If don't need the lines perhaps we should remove them instead?